web/src/components/Navbar/navbar.css

```diff
@@ -1,58 +1,96 @@
-.sticky-nav {
-    top: 0;
-    width: 100% !important;
-    border-bottom: var(--borderline);
-    background: var(--background);
+:root {
```

Comment:
since this branch seems to have the global.css and navbar.css imported into main.css, you don't need those root styles =)